rdkit_utils/serial.py

```
@@ -206,15 +207,27 @@ def _get_mols(self):
         A generator yielding RDKit Mol objects.
         """
         if self.mol_format == 'sdf':
-            return self._get_mols_from_sdf()
+            mols = self._get_mols_from_sdf()
         elif self.mol_format == 'smi':
-            return self._get_mols_from_smiles()
+            mols = self._get_mols_from_smiles()
         elif self.mol_format == 'pkl':
-            return self._get_mols_from_pickle()
+            mols = self._get_mols_from_pickle()
         else:
             raise NotImplementedError('Unrecognized molecule format ' +
                                       '"{}"'.format(self.mol_format))
 
+        # skip read errors
+        while True:
+            try:
+                mol = mols.next()
+            except StopIteration:
+                break
+            except Exception:
```

Should we print a warning here? Silently skipping errors seems like it might cause issues.